core/src/main/resources/struts-default.xml

```diff
@@ -372,6 +374,15 @@
                  associated with requests that should not be validated.
                  -->
             <interceptor-stack name="defaultStack">
+                <interceptor-ref name="coepInterceptor">
```

Comment:
we've discussed a few times that these interceptors should not be at the top of the stack. If you still have some questions about why this is the case, I'd be happy to clarify it.